griptape/tasks/image_query_task.py

```diff
@@ -49,8 +47,11 @@ def input(self) -> tuple[TextArtifact, list[ImageArtifact]]:
     @input.setter
     def input(
         self,
-        value: tuple[TextArtifact, list[ImageArtifact]]
-        | Callable[[BaseTask], tuple[TextArtifact, list[ImageArtifact]]],
+        value: (
+            tuple[str, list[ImageArtifact]]
+            | tuple[TextArtifact, list[ImageArtifact]]
+            | Callable[[BaseTask], ListArtifact]
```

Comment:
My one consideration was whether or not it's useful to accept a `ListArtifact` here directly, similarly to how the `Callable` now just returns a `ListArtifact`. I don't think I've run into a situation where this would have made anything more convenient, and these input type unions are already unwieldy enough, but curious what you think.